content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/labels.rst

```diff
@@ -26,14 +27,45 @@ labels. Finally, provide the company's :ref:`source address
    :align: center
    :alt: Set the "Get Rate and Create Shipment" option.
 
+.. _inventory/shipping_receiving/picking-config:
+
+Labels for multi-step
+---------------------
+
+For companies using :doc:`two <../daily_operations/receipts_delivery_two_steps>` or :doc:`three step
+delivery <../daily_operations/delivery_three_steps>`, labels can be triggered to print after
+validating the picking or packing operation. To do that, go to :menuselection:`Inventory app -->
+Configuration --> Operations Types`, and choose the desired operation.
+
+On the :guilabel:`Operation Type` configuration page, tick the :guilabel:`Print Label` checkbox.
+Enabling this feature ensures that the third-party shipping label is printed upon validating this
+operation.
+
+.. example::
+   For :doc:`two-step delivery <../daily_operations/receipts_delivery_two_steps>`, where products
+   are placed directly in packages during picking, companies can print shipping labels during
+   picking instead of delivery. Odoo allows users to enable the :guilabel:`Print Label` feature on
+   the `Pick` operation itself to achieve this flexibility.
+
+   .. image:: labels/pick-print-label.png
+      :align: center
+      :alt: Enable the "Print Label" feature.
+
 Print tracking labels
 =====================
 
-Tracking labels are generated after the delivery order (DO) is validated.
+To configure automatic printing of tracking labels, add the third-party carrier to the sales order
+(SO) in the *Sales* app. Then, validate the delivery order (DO), or the relevant operation, to
+trigger the label to be automatically printed.
```

Comment:
This section seems to be both redundant and in conflict with lines 66-71. 
Consider condensing in a way where all of the conditions are covered, but in one paragraph, with one clear set of instructions for each condition.

Reply:
Addressed this comment with the following revision (:

```rst
 Tracking labels are printed when specific operations are validated. By default, validating a
delivery order (DO) generates a tracking label in the chatter.

.. note::
   For companies using two or three step delivery, refer to the :ref:`printing labels for multi-step
   delivery <inventory/shipping_receiving/picking-config>` section to learn how to print the label
   after validating a picking or packing operation.

When both the *Sales* and *Inventory* apps are installed, begin in the :menuselection:`Sales` app,
and proceed to the desired quotation or sales order (SO). There, and :ref:`add the shipping cost
<inventory/shipping_receiving/add-shipping-quote>` to the order. Then, navigate to the linked |DO| —
or another operation type when using multi-step delivery — to validate the operation and print the
label.

If only the *Inventory* app is installed, create :abbr:`DOs (Delivery Orders)` directly in the
:menuselection:`Inventory` app , :ref:`add the third-party carrier
<inventory/shipping_receiving/validate-print-label>` in the :guilabel:`Carrier` field, and validate
the |DO|.
```